### PR TITLE
added requested questions

### PR DESCRIPTION
**Pull request by codybousc**
I added questions, requested by the site admin, for potential Hacker Hours hosts to consider. Request seen here: https://github.com/afeld/hackerhours.org/issues/38

### CODE REVIEW

File: diy-guide.md [hunk 1]
```diff
@@ -4,12 +4,21 @@ title: DIY Guide
...
 * not a class, and not a lecture
 
+Here are a few things to take into account when considering hosting:
+
+* Why would you like to host Hacker Hours?
+* Do you haven any specific goals/objectives for hosting? (recruiting, brand recognition, giving back, etc.) <br />
+* What are you thinking in terms of frequency? <br />
+* What is the space like? <br />
+* How many attendees can your space hold comfortably? <br />
+* How many folks from your organization do you expect to stay for each event, on average? How many of them our technical? <br />
```
Comment (afeld) [inline diy-guide.md:20]: Mind removing the `<br />`s? Not actually necessary in unordered lists in Markdown.

Comment (codybousc) [inline diy-guide.md:20]: Yikes. I deleted the code from my desktop and can't seem to make this happen without doing another pull request!

Comment (afeld) [inline diy-guide.md:20]: FYI, one of the big perks of Git is that your computer can explode, and you can pick up your work right where you left off! Should you run into the problem again:

``` bash
# clone the repository from your fork
git clone https://github.com/codybousc/hackerhours.org.git
# go into the directory
cd hackerhours.org
# switch to this branch
git checkout diy-questions

# ...make changes...

# commit the changes
git commit -a -m "removed the extraneous <br> tags"
# send them up to github
git push -u origin diy-questions
# the pull request will now be updated
```

**Comment by afeld** [comment]
Moved to #140.

File: diy-guide.md (hunk 1 above)
Comment (afeld) [inline diy-guide.md:20]: ...or the equivalent in GitHub Desktop, or whatever tool you use.
